CRM/Contribute/Page/ContributionRecurPayments.php

```diff
@@ -45,6 +45,7 @@ private function loadRelatedContributions() {
       'contribution_recur_id' => $this->id,
       'contact_id' => $this->contactId,
       'options' => array('limit' => 0),
+      'contribution_test' => '',
```

Comment:
Been bitten by this recently in two of my payment processors!  So setting it to `''` will retrieve both live and test then?

Reply:
Apparently so!